app/src/main/java/com/example/next_show/data/RecommendationClient.java

```diff
@@ -25,13 +26,37 @@
     public static final int FORBIDDEN_REQUEST = 403;
     public static final int LIMIT = 5;
 
+    // instance vars
+    private Shows showsObj;
+    private FeedFragment.GenreMatchedCallback genreMatchedCallback;
+    private FeedFragment.ShowCallback showCallback;
```

Comment:
Ah, so these shouldn't need to be instance variables.  You're just going to pass in the callbacks when the method is called